src/db/connection.js

```diff
@@ -0,0 +1,20 @@
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
+dotenv.config();
+
+export const db_connections =()=>{
+    const db_user = process.env.USER;
+    const db_password = process.env.PWD;
+    const db_host = process.env.HOSTNAME;
```

Comment:
I would like to suggest that you use dash between two words in the same variable

```suggestion
    const db_host = process.env.HOST_NAME;
```